docs/guides/abci/forum-application/2.intro-forumApp.md

```diff
@@ -0,0 +1,37 @@
+# Forum Application
+
+In this tutorial, we will build a forum application using `ABCI++`. We will demonstrate the use of `ABCI++` functions like - `CheckTx`, `PrepareProposal`, `ProcessProposal`, `FinalizeBlock` and `VoteExtension`
```

Comment:
`VoteExtension`: do you mean ExtendVote?

Reply:
modified